### PR TITLE
[v7r2] Use selectors module

### PR DESCRIPTION
**Pull request by chrisburr**
closes https://github.com/DIRACGrid/DIRAC/issues/5232

BEGINRELEASENOTES

*Core
FIX: Use selectors(2) instead of select.select to avoid issues with file descriptors > 1024

ENDRELEASENOTES

### CODE REVIEW

File: src/DIRAC/Core/DISET/private/MessageBroker.py [hunk 1]
```diff
@@ -7,11 +7,16 @@
 __RCSID__ = "$Id$"
 
 import threading
-import select
 import time
+import select
```
Comment (fstagni) [inline src/DIRAC/Core/DISET/private/MessageBroker.py:11]: It does not seem needed anymore.

**Review by fstagni — COMMENTED** [review]
Have you hotfixed somewhere with this?

File: src/DIRAC/Core/DISET/private/MessageBroker.py (hunk 1 above)
Comment (chrisburr) [reply src/DIRAC/Core/DISET/private/MessageBroker.py:11]: There is still the `except select.error`. I suspect it shouldn't be there but I'm not confident enough to remove it.

**Comment by chrisburr** [comment]
> Have you hotfixed somewhere with this?

Sorry missed this question, yes I have on lbvobox306.

**Review by fstagni — APPROVED** [review]

**Pull request merged**